frontend/test/lit/test_from_plxpr.py

```diff
@@ -338,3 +338,27 @@ def circuit(w1: int, w2: int, w3: int):
 
 
 test_multi_qubit_gates_on_different_dynamic_wires()
+
+
+def test_pass_application():
+    """Application of pass decorators."""
+
+    dev = qml.device("null.qubit", wires=1)
+
+    qml.capture.enable()
```

Comment:
Can we use the `use_capture` test fixture here? (If so, we can also remove the `qml.capture.disable()` at the end).

```suggestion
@pytest.mark.usefixtures("use_capture")
def test_pass_application():
    """Application of pass decorators."""

    dev = qml.device("null.qubit", wires=1)
```

Reply:
This is lit test and doesn't see the fixtures

Reply:
Ah, my mistake. Never mind then.